extensions/ip_cloaking_hmac.c

```diff
@@ -0,0 +1,187 @@
+/**
+ * Cloaks a users hostname using HMAC-MD5.
+ *
+ * Copyright (C) 2015 Patrick Godschalk
+ *
+ * This file is part of charybdis.
+ *
+ * charybdis is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * charybdis is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with charybdis.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <openssl/hmac.h>
+#include "stdinc.h"
+#include "modules.h"
+#include "hook.h"
+#include "client.h"
+#include "ircd.h"
+#include "send.h"
+#include "hash.h"
+#include "s_conf.h"
+#include "s_user.h"
+#include "s_serv.h"
+#include "numeric.h"
+#include "newconf.h"
+
+// This could be a config option, but since cloaking is not intended
+// for proper security to begin with, we don't really care.
```

Comment:
it *is* defined as a config option later, in _modinit(), no? if so, why is this comment there?